### PR TITLE
refactor for remarks pill

### PR DESCRIPTION
**Pull request by SidelineCory24**
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2267)

To-do

- [x] send more remarks data for aih to FE

### CODE REVIEW

File: talentmap_api/fsbid/services/common.py [hunk 1]
```diff
@@ -772,14 +772,15 @@ def convert_to_fsbid_ql(filters):
 
 
 def categorize_remark(remark = ''):
-    obj = { 'title': remark, 'type': None }
+    obj = { 'text': remark, 'type': None }
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/common.py:775]: 🙌

File: talentmap_api/fsbid/services/common.py [hunk 2]
```diff
@@ -789,7 +790,17 @@ def parse_agenda_remarks(remarks_string = ''):
...
+    remarks_values = []
+    for value in values:
+        for remark in ai_remarks:
+            if remark['text'] == value['text']:
+                remarks_values.append(remark)
+        if value['type'] == 'person':
+            remarks_values.append(value)
+    
+    final_remarks = remarks_values if len(remarks_values) else values
+    return final_remarks
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/common.py:803]: we'll need to refactor, but for now we can do 

```
remarks_values = []
for value in values:
    if pydash.find(ai_remarks, {'text': value['text']}):
        remarks_values.append({**value, **pydash.find(ai_remarks, {'text': value['text']})})
    if value['type'] == 'person':
        remarks_values.append(value)

return remarks_values
```

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/common.py:803]: the find saves us a loop

File: talentmap_api/fsbid/services/agenda.py [hunk 3]
```diff
@@ -41,12 +41,13 @@ def get_agenda_items(jwt_token=None, query = {}, host=None):
...
     '''
     from talentmap_api.fsbid.services.agenda_employees import get_agenda_employees
+    remarks = get_agenda_remarks({}, jwt_token)
     args = {
         "uri": "",
         "query": query,
         "query_mapping_function": convert_agenda_item_query,
         "jwt_token": jwt_token,
-        "mapping_function": fsbid_single_agenda_item_to_talentmap_single_agenda_item,
+        "mapping_function": partial(fsbid_single_agenda_item_to_talentmap_single_agenda_item, remarks=remarks),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:50]: 🥳

File: talentmap_api/fsbid/services/common.py (hunk 2 above)
Comment (SidelineCory24) [reply talentmap_api/fsbid/services/common.py]: done

**Review by elizabeth-jimenez — APPROVED** [review]
😸

**Pull request merged**